config/charts/knative-operator/templates/operator.yaml

```diff
@@ -4875,6 +4875,13 @@ spec:
                                       type: string
                                   type: object
                                 tls:
```

Comment:
I am wondering whether you also need to change `https://github.com/knative/operator/blob/main/config/charts/knative-operator/values.yaml` for the default values of these new fields.

Reply:
These are changes in Knative Serving CRDs so I do not think there will be any change in values.yaml of knative-operator helm chart. 
These changes will allow for httpRedirect configuration while creating Knative Serving Custom Resource after installing knative-operator through this helm chart. What do you think?